apps/platform-app/src/app/[locale]/(auth)/(components)/LogOutBtn/LogOutBtn.tsx

```diff
@@ -16,7 +20,7 @@ const LogOutBtn = () => {
       disabled={isCalled}
       onClick={() => {
         setIsCalled(true)
-        signOut({ callbackUrl: AppRoutes.home })
+        onClick ? onclick : signOut({ callbackUrl: AppRoutes.home })
```

Comment:
**issue:** +1 what Nerdy said

Reply:
Ohh, that was a dangerous misspelling, thanks for catching that.